lib/_stream_readable.js

```diff
@@ -302,7 +302,8 @@ function chunkInvalid(state, chunk) {
 }
 
 
-// if it's past the high water mark, we can push in some more.
+// It simply checks the the state length with high water mark
+// which will return true if we are below the watermark
 // Also, if we have no data yet, we can stand some
```

Comment:
Missing period?

Reply:
Sorry but I don't get it how is it missing period

Reply:
no it isn't missing period made it more clear in the latest commit